x/chainlet/keeper/msg_server_disable_stack_version.go

```diff
@@ -20,8 +19,12 @@ func (k msgServer) DisableChainletStackVersion(goCtx context.Context, msg *types
 
 	p := k.GetParams(ctx)
 	if p.ChainletStackProtections {
-		dacAddr := dactypes.NewAddress(dactypes.AddressFormat_ADDRESS_BECH32, msg.Creator)
-		if !k.dacKeeper.Allowed(ctx, dacAddr) {
+		var addr sdk.AccAddress
+		addr, err = sdk.AccAddressFromBech32(msg.Creator)
+		if err != nil {
+			return
+		}
+		if !k.aclKeeper.Allowed(ctx, addr) {
```

Comment:
A slightly side note but I think the error message below should have `to disable` instead of `to modify`.